test/Validate.Capability.cpp

```diff
@@ -40,18 +40,77 @@
 namespace {
 
 using spvtest::ScopedContext;
+using std::get;
 using std::make_pair;
+using std::ostringstream;
 using std::pair;
 using std::string;
-using std::stringstream;
 using std::tuple;
 using std::vector;
 using testing::Combine;
 using testing::Values;
 using testing::ValuesIn;
 
-using ValidateCapability =
-    spvtest::ValidateBase<tuple<string, pair<string, vector<string>>>>;
+// Parameter for validation test fixtures.  Captures assembly to test in two
+// parts: a variable top line and a fixed remainder.  The top line will be an
+// OpCapability instruction, while the remainder will be some assembly text that
```

Comment:
Well, this is the comment for `CapTestParameter`, right? And the very beginning sentence reads like we are starting to explain what `Parameter for validation test fixtures`, right?